flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/calcite/FlinkCalciteSqlValidatorTest.java

```diff
@@ -63,6 +64,55 @@ void testInsertInto2() {
                 .hasMessageContaining(" Number of columns must match number of query columns");
     }
 
+    @Test
+    void testInsertInto3() {
```

Comment:
Tests should have a descriptive name. E.g. `testInsertIntoValuesColumnsMismatch`